src/App.vue

```diff
@@ -37,9 +43,36 @@ export default defineComponent({
     usePoolCreationWatcher();
     useGlobalQueryWatchers();
     useGnosisSafeApp();
-    const { isWalletSelectVisible, toggleWalletSelectModal } = useWeb3();
+    const {
+      isWalletSelectVisible,
+      toggleWalletSelectModal,
+      isMainnet
+    } = useWeb3();
     const route = useRoute();
     const store = useStore();
+    const router = useRouter();
+    const { addAlert } = useAlerts();
+    const { t } = useI18n();
+
+    // Temporary feature alert for Balancer boosted pools.
+    if (isMainnet.value) {
+      const featureAlert: Alert = {
+        id: 'boosted-pools',
+        priority: AlertPriority.LOW,
+        label: t('alerts.boostedPools'),
+        type: AlertType.FEATURE,
+        actionOnClick: true,
+        action: () =>
+          router.push({
```

Comment:
@garethfuller do you think it worths removing this alert after clicking the banner?

If so, you want to use `removeAlert('boosted-pools');` after the route change.

Reply:
I think it's ok to keep showing it, for now, to make it sticky.